### PR TITLE
test: KUBESAW-236 update ci scripts

### PR DESCRIPTION
**Pull request by rsoaresd**
# Description
-  clean `OWNER_AND_BRANCH_LOCATION`
- cleanup the pull mechanism in `scripts/ci/manage-host-operator.sh` and `scripts/ci/manage-member-operator.sh` (thanks @MatousJobanek)
- delete `scripts/ci/collect-logs.sh` since we do not use it

P.S. Once this PR is merged, I will delete `scripts/ci` folder on toolchain-cicd

## Issue ticket number and link
[KUBESAW-236](https://issues.redhat.com/browse/KUBESAW-236)

### CODE REVIEW

**Comment by rsoaresd** [comment]
/retest

```
    proxy_test.go:385: 
        	Error Trace:	/go/src/github.com/codeready-toolchain/toolchain-e2e/test/e2e/parallel/proxy_test.go:385
        	Error:      	Received unexpected error:
        	            	configmaps "kube-root-ca.crt" not found
        	Test:       	TestProxyFlow/compliant.username/get_resource_from_namespace_outside_of_user's_workspace/success_when_accessing_shared_namespace/using_explicit_workspace_context
```

First time I came across this. Maybe a flaky test (?)

File: scripts/ci/manage-operator.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ else
 fi
 
 WAS_ALREADY_PAIRED_FILE=/tmp/toolchain_e2e_already_paired
-OWNER_AND_BRANCH_LOCATION=${OWNER_AND_BRANCH_LOCATION:-codeready-toolchain/toolchain-cicd/master}
+OWNER_AND_BRANCH_LOCATION=${OWNER_AND_BRANCH_LOCATION:-codeready-toolchain/toolchain-e2e/master}
```
Comment (MatousJobanek) [inline scripts/ci/manage-operator.sh:10]: you can drop this one too and simplify the related section to something like this:
```
start_collecting_logs() {
  scripts/ci/collect-logs.sh -n ${NAMESPACE} &
  scripts/ci/collect-logs.sh -n "openshift-operator-lifecycle-manager"  &
}
```
but TBH, the collect-logs.sh is not really used. I can give you an explanation about it during our 1-1

Comment (rsoaresd) [reply scripts/ci/manage-operator.sh:10]: Deal, thanks!

Comment (MatousJobanek) [reply scripts/ci/manage-operator.sh:10]: Yes, good catch :+1: @mfrancisc

Comment (rsoaresd) [reply scripts/ci/manage-operator.sh:10]: Yes, thanks!! I saw that we need to update `run_wait_until_is_installed` as well. I am just testing locally (finally i am not getting `bad record MAC` anymore today 🙏 )

**Review by mfrancisc — APPROVED** [review]
Nice Job 👍 

Thanks for simplifying and cleaning this up 🙏

**Review by MatousJobanek — APPROVED** [review]
cool :+1: Thanks :1st_place_medal:

**Pull request merged**